### PR TITLE
feat(ssh): add new section: ssh keys

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import some from 'lodash/some';
+
+const MESSAGES_CONTAINER_NAME = 'pci.projects.project.sshKeys';
+
+export default class ProjectSshKeysController {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, OvhApiCloudProjectSshKey, projectId) {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js]: Alphabetical order maybe

File: packages/manager/modules/pci/src/projects/project/ssh-keys/add/add.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <oui-modal
+        data-heading="{{:: 'pci_projects_project_sshKeys_add_title' | translate}}"
+        data-primary-action="$ctrl.add()"
+        data-primary-label="{{:: 'pci_projects_project_sshKeys_add_submit_label' | translate }}"
+        data-primary-disabled="$ctrl.isLoading || pci_projects_project_sshKeys_add_key.$invalid"
+        data-secondary-action="$ctrl.goBack()"
+        data-secondary-label="{{:: 'pci_projects_project_sshKeys_add_cancel_label' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isLoading">
+        <oui-field label="{{ ::'pci_projects_project_sshKeys_add_name' | translate }}" size="xl">
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/add/add.html]: According to the oui-modal implementation there are some missing `data-` prefixes inside this template.

File: packages/manager/modules/pci/src/projects/project/ssh-keys/add/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-api-wrappers'; // should be a peer dependency of ovh-api-services
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/add/index.js]: Shouldn't be injected to the module dependency list?

File: packages/manager/modules/pci/src/projects/project/ssh-keys/add/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "pci_projects_project_sshKeys_add_title": "Ajouter une clé SSH",
+  "pci_projects_project_sshKeys_add_submit_label": "Ajouter",
+  "pci_projects_project_sshKeys_add_cancel_label": "Annuler",
+  "pci_projects_project_sshKeys_add_success": "La clé SSH a été ajouté avec succès",
+  "pci_projects_project_sshKeys_add_error": "Une erreur est survenue lors de l'ajout de la clé SSH: {{ error }}",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/add/translations/Messages_fr_FR.json]:
```suggestion
  "pci_projects_project_sshKeys_add_error": "Une erreur est survenue lors de l'ajout de la clé SSH : {{ error }}",
```

File: packages/manager/modules/pci/src/projects/project/ssh-keys/index.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-api-wrappers'; // should be a peer dependency of ovh-api-services
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/index.js]: Shouldn't be injected to the module dependency list?

File: packages/manager/modules/pci/src/projects/project/ssh-keys/remove/index.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-api-wrappers'; // should be a peer dependency of ovh-api-services
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/remove/index.js]: Shouldn't be injected to the module dependency list?

File: packages/manager/modules/pci/src/projects/project/ssh-keys/remove/remove.controller.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    this.$state = $state;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.OvhApiCloudProjectSshKey = OvhApiCloudProjectSshKey;
+
+    this.serviceName = $stateParams.projectId;
+    this.keyId = $stateParams.keyId;
+    this.isLoading = false;
+
+    this.$onInit();
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/remove/remove.controller.js]: Is it needed to call the `$onInit` hook inside the constructor class? Shouldn't be the controller's instantiation which trigger this hook?

File: packages/manager/modules/pci/src/projects/project/ssh-keys/remove/translations/Messages_fr_FR.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_sshKeys_remove_title": "Supprimer une clé SSH",
+  "pci_projects_project_sshKeys_remove_description": "Cette action est irréversible. Voulez-vous supprimer la clé {{ name }} ?",
+  "pci_projects_project_sshKeys_remove_submit_label": "Supprimer",
+  "pci_projects_project_sshKeys_remove_cancel_label": "Annuler",
+  "pci_projects_project_sshKeys_remove_success": "La clé SSH a été supprimée avec succès.",
+  "pci_projects_project_sshKeys_remove_error": "Une erreur est survenue lors de la suppression de la clé SSH: {{ error }}."
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/remove/translations/Messages_fr_FR.json]:
```suggestion
  "pci_projects_project_sshKeys_remove_error": "Une erreur est survenue lors de la suppression de la clé SSH : {{ error }}."
```

File: packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.OvhApiCloudProjectSshKey = OvhApiCloudProjectSshKey;
+    this.projectId = projectId;
+
+    this.messageHandler = CucCloudMessage.subscribe(MESSAGES_CONTAINER_NAME, {
+      onMessage: () => this.refreshMessage(),
+    });
+
+    this.$onInit();
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js]: Is it needed to call the `$onInit` hook inside the constructor class?

File: packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  $onInit() {
+    this.OvhApiCloudProjectSshKey
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js]: According to the previous review, that promise shouldn't be returned too? 

```suggestion
    return this.OvhApiCloudProjectSshKey
```

File: packages/manager/modules/pci/src/projects/project/ssh-keys/add/index.js (hunk 3 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/project/ssh-keys/add/index.js]: > // should be a peer dependency of ovh-api-services

So it's ovh-api-services which required it :)

File: packages/manager/modules/pci/src/projects/project/ssh-keys/remove/remove.controller.js (hunk 7 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/project/ssh-keys/remove/remove.controller.js]: Only for component, will do it in next PR ;)

**Pull request merged**